encoding/json/decode.go

```diff
@@ -866,6 +872,28 @@ func (d *Decoder) decodeEnum(valueJSON any) cadence.Enum {
 	))
 }
 
+func (d *Decoder) decodeInclusiveRange(valueJSON any) cadence.InclusiveRange {
+	obj := toObject(valueJSON)
+
+	start := obj.GetValue(d, startKey)
+	end := obj.GetValue(d, endKey)
+	step := obj.GetValue(d, stepKey)
+
+	value := cadence.NewMeteredInclusiveRange(
+		d.gauge,
+		start,
+		end,
+		step,
+	)
+
+	// TODO: Ensure that start, end and step have the same type.
```

Comment:
maybe add some tests in `TestRuntimeScriptParameterTypeValidation`, for importing range values with mixed types.

Reply:
Done, added test cases and removed the TODO.